resources/services/workspaces/directories.go

```diff
@@ -0,0 +1,243 @@
+package workspaces
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/workspaces"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource directories --config gen.hcl --output .
+func Directories() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_workspaces_directories",
+		Description:  "Describes a directory that is used with Amazon WorkSpaces.",
+		Resolver:     fetchWorkspacesDirectories,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("workspaces"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"id"}},
+		Columns: []schema.Column{
```

Comment:
Can you add an ARN field?
https://docs.aws.amazon.com/workspaces/latest/adminguide/workspaces-access-control.html